<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>FoLST2021.knit</title>

<script src="index_files/header-attrs-2.11/header-attrs.js"></script>






<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Special Elite' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 23 35;
margin: 0;
padding: 0;
}
body {
margin: 0px;
padding: 0px;
width: 23;
height: 35;
text-align: justify;
font-size: 45px;
line-height: 1.05;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #00808024;
color: #0b4545;
padding: 1.2mm;
line-height: 1;
border-radius: 2mm;
}
caption {
margin-bottom: 10px;
font-size: 20pt;
font-style: italic;
}

tbody tr:nth-child(odd) {
    background-color: #00808020;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  font-size: 40%;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 1em;
  padding-left: 1em;
  line-height: 1em;
}
table {
  margin: auto;
}
th {
  padding-left: 5mm;
  padding-right: 5mm;
}
.caption {
font-size: 20pt;
font-style: italic;
padding-top: 0;
}
.references {
font-size: 20px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 23;
height: calc(35 *  (1 - 1 - 1 - 0.01) );
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
column-gap: 0;
padding-left: 0cm;
padding-right: 0cm;
/* -webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%; */
-webkit-column-rule-style: none;
-moz-column-rule-style: none;
column-rule-style: none;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
background-color: #ffffff;
font-family: Rasa;
margin-top: calc(35 *  1 );
padding-top: 1em;
padding-bottom: 1em;
}
span.citation {
  color: #008080;
  font-weight: bold;
}
a {
text-decoration: none;
color: #008080;
}
#title {
font-size: 125pt;
text-align: left;
margin: 0;
line-height: 98%;
border-bottom: 0;
font-weight: normal;
background: 0;
}
#author {
color: #0b4545;
margin: 0;
line-height: 85%;
font-size: 1.17em;
}
#affiliation {
padding-top: 0.1em;
color: ;
font-style: italic;
font-size: 25px;
margin: 0;
}
sup {
color: #cc0000;
}
.affiliation sup {
font-size: 20px;
}
.author {
text-align: left;
}
.author sup {
font-size: 30px;
}
.author_extra {
color: #008080;
margin: 0;
line-height: 85%;
font-size: 35px;
text-align: left;
}
.outer h1, h2, h3, h4, h5, h6 {
text-align: center;
margin: 0;
font-weight: bold;
}
.section h1 {
  text-align:center;
  padding-bottom:5px;
  background:
    linear-gradient(
      to left,
      #ffffff 1%,
      #ffffff 20%,
      #0b454575 33%,
      #0b4545 50%,
      #0b454575 66%,
      #ffffff 80%,
      #ffffff 99%
    )
    left
    bottom
    #ffffff
    no-repeat;
  background-size:100% 5px ;
  margin-top: 0.5em;
  margin-bottom: 0.5em;
}
.outer h2 {
text-align: center;
}
.outer p, .level2 {
color: #000000;
}
.outer ol {
padding-left: 8%;
padding-right: 8%;
text-align: left;
}
.main {
width: 23;
height: calc(35 * 1);
position: absolute;
background-color: #0b4545;
color: #ffffff90;
font-family: Special Elite;
background-image: linear-gradient(#0b4545 50%,#008080);
}
.main strong {
color: #ffffff;
}
.main strong > sup {
color: #ffffff;
}
.main sup {
color: #ffffff90;
}
#main-img-left {
width: 10%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
}
#main-img-center {
width: 10%;
left: calc(23 * 0.45);
bottom: 0.5in;
position: absolute;
}
#main-img-right {
width: 10%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
}
.main p {
font-size: 150px;
font-family: Special Elite;
text-align: center;
margin: 0;
position: absolute;
top: 50%;
-ms-transform: translateY(-50%);
transform: translateY(-50%);
margin-left: 1em;
}
.fab {
color: #00000030;
font-size: 25px;
}
.twitter, i {
color: #00000030;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000030;
font-size: 35px;
}
.envelope {
color: #00000030;
font-size: 5px;
text-decoration: none;
}
.poster_wrap {
width: 23;
height: 35;
padding: 0cm;
}
.main_bottom {
width: 23;
height: calc(35 * 1);
margin-top: calc(35 * (1 - 1));
position: absolute;
background-color: #0b4545;
background-image: linear-gradient(#008080 10%, #0b4545);
}
.section {
  padding-left: 10mm;
  padding-right: 10mm;
}
span > #tab:mytable {
  font-weight: bold;
}
.orcid img {
  width: 3%;
}
.emphasis {
  background-color: #008080;
  color: #ffffff;
  border: solid #0b2045 3mm;
  margin: 1em;
  padding-left: 0;
  padding-right: 0;
}
.emphasis h1 {
  font-weight: bold;
  background: none;
  background-color: #0b2045;
  padding-bottom: 5mm;
  padding-top: 1mm;
  margin-top: -1mm;
  margin-right: -1mm;
  margin-left: -1mm;
}
.emphasis blockquote {
  border: 0;
}
.emphasis ol {
  padding: 0;
  padding-left: 8%;
  font-size: 100%;
  font-weight: bold;
}
.emphasis p {
  color: #ffffff;
}
</style>
</head>
<body>


<div class="poster_wrap">

<div class="column outer">
<div class="section">

<h3 id="author" class="author">

    </h3>

<h5 id="author_extra", class="author_extra">
Tao Wu<sup>1, 2, 3, <a class="orcid" href="https://orcid.org/0000-0002-8999-9628"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>
 Guangshuai Wang<sup>1, 2, 3</sup>
 Xuan Wang<sup>1, 2, 3</sup>
 Shixiang Wang<sup>1, 2, 3</sup>
 Xue-Song Liu<sup>1,4, <a class="orcid" href="https://orcid.org/0000-0002-7736-0077"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup>
</h5>


<p id="affiliation" class="affiliation">
<sup>1</sup> School of Life Science and Technology, ShanghaiTech University, Shanghai 201203, China<br> <sup>2</sup> Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, Shanghai, China<br> <sup>3</sup> University of Chinese Academy of Sciences, Beijing, China<br> <sup>4</sup> Corresponding Author
</p>
</div>

<div id="abstract" class="section level1">
<h1>Abstract</h1>
<p>Immunoediting, which includes three temporally distinct stages, termed elimination, equilibrium, and escape, has been proposed to explain the interactions between cancer cells and the immune system during the evolution of cancer. However the status of immunoediting in cancer remain unclear, and the existence of neoantigen depletion signal in untreated cancer has been debated. Here we developed a distribution pattern based method for quantifying neoantigen mediated negative selection in cancer evolution. Our method provides a robust and reliable quantification for immunoediting signal in individual cancer patient. The prevalence of immunoediting signal in immunotherapy untreated cancer genome has been demonstrated with this method. Importantly, the elimination and escape stages of immunoediting can be quantified separately, tumor types with strong immunoediting-elimination tend to have weak immunoediting-escape signal, and vice versa. Quantified immunoediting-elimination signal predicts cancer immunotherapy clinical response. Immunoediting quantification provides an evolutional perspective for evaluating the immunogenicity of neoantigen, and reveals potential biomarker for cancer precision immunotherapy.</p>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure1"></span>
<img src="Fig/Fig1.png" alt="Conceptual framework for the quantification of elimination and escape phases of immunoediting." width="80%" />
<p class="caption">
Figure 1: Conceptual framework for the quantification of elimination and escape phases of immunoediting.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure2"></span>
<img src="Fig/Fig2.png" alt="Distribution pattern based method for the quantification of neoantigen mediated negative selection in cancer evolution." width="80%" />
<p class="caption">
Figure 2: Distribution pattern based method for the quantification of neoantigen mediated negative selection in cancer evolution.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure3"></span>
<img src="Fig/Fig3.png" alt="Pan-cancer distributions and features of the quantified immunoediting signals (ESCCF and ESRNA)." width="100%" />
<p class="caption">
Figure 3: Pan-cancer distributions and features of the quantified immunoediting signals (ESCCF and ESRNA).
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure4"></span>
<img src="Fig/Fig4.png" alt="Immunoediting-elimination signal (ESCCF) and neoantigen-mediated negative selection strength quantification." width="100%" />
<p class="caption">
Figure 4: Immunoediting-elimination signal (ESCCF) and neoantigen-mediated negative selection strength quantification.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure5"></span>
<img src="Fig/Fig5.png" alt="Quantified immunoediting-elimination signal (ESCCF) predicts cancer immunotherapy clinical response." width="100%" />
<p class="caption">
Figure 5: Quantified immunoediting-elimination signal (ESCCF) predicts cancer immunotherapy clinical response.
</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<ul>
<li>Developed a brand new method for reliably quantifying neoantigen mediated immunoediting in individual cancer patient.</li>
<li>With the new analysis framework, we demonstrate the pan-cancer existence of neoantigen mediated negative selection signal.</li>
<li>Elimination and escape stages of immunoediting can be quantified separately,
tumor types with strong immunoediting-elimination tend to have weak immunoediting-escape signal, and vice versa.</li>
<li>Quantified immunoediting-elimination signal predicts cancer immunotherapy clinical response.</li>
</ul>
</div>
<div id="more" class="section level1">
<h1>More</h1>
<p>You can scan QR code at bottom center to see online analysis report. All code and related data are published at <a href="https://github.com/XSLiuLab/tumor-immunogenicity-score" class="uri">https://github.com/XSLiuLab/tumor-immunogenicity-score</a>.</p>
</div>
<div id="acknowledgement" class="section level1">
<h1>Acknowledgement</h1>
<p>We thank the authors and participating patients of immunotherapy publications for providing the data for this analysis. Our gratitude is also extended to the TCGA project for making cancer genomics data available for analysis. Thank ShanghaiTech University High Performance Computing Public Service Platform for computing services. Thanks also to other members of Liu lab for helpful discussion.</p>
</div>
<div id="recent-works-of-our-lab" class="section level1">
<h1>Recent works of our lab</h1>
<ul>
<li>He, Zaoke, Sun, Xiaoqin, Wang, Shixiang, et al. “Ggct (g-glutamyl cyclotransferase) plays an important role in erythrocyte antioxidant defense and red blood cell survival.” <strong>BRITISH JOURNAL OF HAEMATOLOGY</strong> (2021).</li>
<li>Wang Shixiang, et al. “UCSCXenaShiny: An R/CRAN Package for Interactive Analysis of UCSC Xena Data.” <strong>BIOINFORMATICS</strong> (2021).</li>
<li>Wang Shixiang, et al. “Copy number signature analysis tool and its application in prostate cancer reveals distinct mutational processes and clinical outcomes.” <strong>PLOS GENETICS</strong> (2021).</li>
<li>He, Zaoke, Wu, Tao, Wang, Shixiang, et al. “Pan-cancer noncoding genomic analysis identifies functional CDC20 promoter mutation hotspots.” <strong>ISCIENCE</strong> (2021).</li>
<li>Wang Shixiang, et al. “Sigflow: an automated and comprehensive pipeline for cancer genome mutational signature analysis.” <strong>BIOINFORMATICS</strong> (2020).</li>
<li>Wang, Shixiang1, He, Zaoke, et al. “Can tumor mutational burden determine the most effective treatment for lung cancer patients?” <strong>LUNG CANCER MANAGEMENT</strong> (2019)</li>
<li>Wang, Shixiang, et al. “Antigen presentation and tumor immunogenicity in cancer immunotherapy response prediction.” <strong>eLife</strong> (2019).</li>
<li>Wang, Shixiang, Li An Cowley, and Xue-Song Liu. “Sex Differences in Cancer Immunotherapy Efficacy, Biomarkers, and Therapeutic Strategy.” <strong>Molecules</strong> 24.18 (2019): 3214.</li>
<li>He, Zaoke, et al. “Ras Downstream Effector GGCT Alleviates Oncogenic Stress.” <strong>iScience</strong> 19 (2019): 256-266.</li>
<li>Wang, Shixiang, and Xuesong Liu. “The UCSCXenaTools R package: a toolkit for accessing genomics data from UCSC Xena platform, from cancer multi-omics to single-cell RNA-seq.” <strong>The Journal of Open Source Software</strong> 4 (2019).</li>
<li>Wang, Shixiang, et al. “The predictive power of tumor mutational burden in lung cancer immunotherapy response is influenced by patients’ sex.” <strong>International journal of cancer</strong> (2019).</li>
<li>Wang, Shixiang, et al. “APOBEC3B and APOBEC mutational signature as potential predictive markers for immunotherapy response in non-small cell lung cancer.” <strong>Oncogene</strong> 37.29 (2018): 3924.</li>
</ul>
<hr />
<p><a href="https://github.com/XSLiuLab/">©Cancer Biology Group</a> 2019</p>
<p><small>Research group led by Xue-Song Liu in ShanghaiTech. University. Lab website is shown in QR code at bottom right.</small></p>
</div>

</div>
<div class="main">
<p><strong>Quantification of neoantigen-mediated immunoediting in cancer evolution</strong></p>
</div>
<div class="main_bottom">
<img id="main-img-left" src=Fig/shanghaitech_logo.png>
<img id="main-img-center" src=Fig/liulab.png>
<img id="main-img-right" src=>
</div>
</div>



</body>
</html>
